robottelo/manifests.py

```diff
@@ -103,10 +105,14 @@ def original(self):
             self._download_manifest_info()
         return six.BytesIO(self.template)
 
-
+if not settings.configured:
+    settings.configure()
 # Cache the ManifestCloner in order to avoid downloading the manifest template
 # every single time.
-_manifest_cloner = ManifestCloner()
+_manifest_cloner = {
```

Comment:
I would use a different approach. I would make `ManifestClone` to have an internal dict where users can call clone passing a manifest key (the same key present on properties file) then it would check for the caching dict if the required manifest was already downloaded and then download if necessary. It also would require that  the original method receive the manifest key (or name) so it could return the proper original manifest.